packages/composites/token-allocation/src/dataProvider.ts

```diff
@@ -2,13 +2,17 @@ import { Requester } from '@chainlink/external-adapter'
 import { PriceAdapter, ResponsePayload } from './types'
 
 const getPrices = (apiConfig: any) => async (
+  jobRunID: string,
   symbols: string[],
   quote: string,
   withMarketCap = false,
 ): Promise<ResponsePayload> => {
   const results = await Promise.all(
     symbols.map(async (base) => {
-      const data = { data: { base, quote, endpoint: withMarketCap ? 'marketcap' : 'price' } }
+      const data = {
+        id: jobRunID,
```

Comment:
Why do we need this `jobRunID` here?

Reply:
`jobRunID` is not sent through automatically now. It will also be useful to get track of it on relying adapters to filter by job